src/Definition/Encoding/PhpArrayToRelationshipCollectionEncoderInterface.php

```diff
@@ -5,8 +5,17 @@
 namespace Undabot\JsonApi\Definition\Encoding;
 
 use Undabot\JsonApi\Definition\Model\Resource\Relationship\RelationshipCollectionInterface;
+use Undabot\JsonApi\Implementation\Encoding\Exception\JsonApiEncodingException;
 
+/** @psalm-suppress PossiblyUnusedMethod */
 interface PhpArrayToRelationshipCollectionEncoderInterface
 {
+    /**
+     * @param array<string,array<string,mixed>> $relationships
+     *
+     * @throws JsonApiEncodingException
+     *
+     * @psalm-suppress PossiblyUnusedMethod
```

Comment:
Yes

Reply:
I removed the annotation because it's also used inside PhpArrayToRelationshipCollectionEncoder. But yes, if it's only used inside some test we need annotation.